api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -74,25 +69,56 @@ public void runBeforeEachTest() {
 			adminService.setImplementationIdHttpClient(implementationHttpClient);
 			cacheManager = Context.getRegisteredComponent("apiCacheManager", CacheManager.class);
 		}
-		
+
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldReturnSystemInformation() {
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
+		assertEquals(5, systemInfoMap.size());
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldAddAllLoadedModulesInformation() {
+		Module module = new Module("name", "moduleId", "packageName", "author", "description", "version");
+		ModuleFactory.loadModule(module, false);
+		int numberOfModules = ModuleFactory.getLoadedModules().size();
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
+		assertEquals(numberOfModules + 1, systemInfoMap.get("SystemInfo.title.moduleInformation").size());
 	}
 	
+	/**
+	 * @see AdministrationService#removeGlobalPropertyListener(GlobalPropertyListener)
+	 */
+	@Test
+	public void removeGlobalPropertyListener_shouldNotFailOnNullListener() {
+		GlobalPropertyListener globalPropertyListener = null;
+		assertNull(globalPropertyListener);
+		adminService.removeGlobalPropertyListener(globalPropertyListener);
+	}
+
 	/**
 	 * Tests the AdministrationService.executeSql method with a sql statement containing a valid
 	 * group by clause
-	 * 
-	 * @see AdministrationService#executeSQL(String,null)
+	 *
+	 * @see AdministrationService#executeSQL(String, boolean) 
```

Comment:
please remove these javadocs (since :) they clutter)

Reply:
should I remove all the javadocs?? or only for the tests I wrote??

Reply:
you can remove the javadocs on the tests you write or change and leave the others as is